site/cypress/e2e/pronouns_change.spec.js

```diff
@@ -0,0 +1,75 @@
+
+describe('Tests cases abut changing user pronouns', () => {
+
+    let oldPronouns = '';
+
+    it('Tests changing pronouns for student', () => {
+        cy.visit('/user_profile');
+        cy.login('student');
+
+        //open pronouns form
+        cy.get('#pronouns_val').click();
+        const e = cy.get('#user-pronouns-change');
+
+        //save old pronouns
+        e.then(($pronounsInput) => {
+            oldPronouns = $pronounsInput.val();
+        });
+
+        //type in new pronouns
+        cy.get('button[aria-label="Clear pronoun input"]').click(); //clear input using trash can
+        e.type('They/Them');
+        cy.get('#edit-pronouns-form .form-button-container > .btn-primary').first().click();
+
+    });
+
+    it('Verifies changed pronouns as instructor in Manage Students', () => {
+        cy.visit(['sample','users']);
+        cy.login('instructor');
+
+        //ensure pronouns column is on
+        cy.get('a[href="javascript:toggleColumnsForm()"]').click(); //open toggle columns form
```

Comment:
It would be better to target this by something other than the name of a JS function.  We don't want to have to change this test if someone refactors the JS and changes the name of the function.